spec/mailers/previews/publishers/job_application_preview.rb

```diff
@@ -0,0 +1,11 @@
+# Documentation: app/mailers/previewing_emails.md
+class Publishers::JobApplicationPreview < ActionMailer::Preview
+  def applications_received
+    publisher = Publisher.all.find { |p| p.vacancies.any? }
+    vacancy = publisher.vacancies.last
+
+    FactoryBot.create(:job_application, vacancy: vacancy, submitted_at: Date.yesterday, status: "submitted")
+
+    Publishers::JobApplicationMailer.applications_received(publisher: Publisher.first)
```

Comment:
I don't quite understand this. We first choose a publisher that has vacancies, then we create a job application for one of those vacancies and on the last line we use `Publisher.first`. Mail preview can assume that the seed has run and there are already submitted job applications, can't we use one of those?

Reply:
Ah yes, `Publisher.first` should be `publisher`. But I can rewrite this to use the seed data.

Reply:
I've just replaced the method with # TODO because it's not a priority right now. I have already generated the previews that were needed by Content Design.